content/en/blog/_posts/2024-11-11-memory-manager-moves-to-ga/index.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: blog
+title: "Memory Manager Goes GA"
+date: 2024-11-11
+slug: memory-manager-goes-ga
+author: >
+  [Talor Itzhak](https://github.com/Tal-or) (Red Hat)
+draft: true
+---
+
+With Kubernetes 1.32, the memory manager has officially graduated to General Availability (GA),
+marking a significant milestone in the journey toward efficient and predictable memory allocation for containerized applications.
+Since Kubernetes v1.22, where it graduated to beta, the memory manager has proved itself reliable, stable and a good complementary feature for the
+[CPU Manager](/docs/tasks/administer-cluster/cpu-management-policies/).
+
+As part of kubelet's workload admission process, 
+the memory manager provides topology hints 
+to optimize memory allocation and alignment. 
+This enables users to allocate exclusive
+memory for `Guaranteed` [QoS class](/docs/tasks/configure-pod-container/quality-service-pod/#qos-classes) pods.
+More details about the process can be found in the memory manager goes to beta [blog](/blog/2021/08/11/kubernetes-1-22-feature-memory-manager-moves-to-beta/).
+
+Most of the changes introduced since the Beta are bug fixes, internal refactoring and 
+observability improvements, such as metrics and better logging.
+
+## Observability improvements
+
+As part of the effort
+to increase the observability of memory manager, new metrics have been added
+to provide some statistics on memory allocation patterns.
+
+
+**memory_manager_pinning_requests_total**
+tracks the number of times the pod spec required the memory manager to pin memory pages.
+
+**memory_manager_pinning_errors_total**
+tracks the number of times the pod spec required the memory manager 
+to pin memory pages, but the allocation failed.
+
+
+## Improving memory manager reliability and consistency
+
+kubelet does not guarantee pod ordering
+when admitting pods after a restart or reboot.
+
+In certain edge cases, this behavior could cause
+the memory manager to reject some pods,
+and in more extreme cases, it may cause kubelet to fail upon restart.
+
+Previously, the beta implementation lacked certain checks and logic to prevent 
+these issues.
+
+To stabilize the memory manager for general availability (GA) readiness,
```

Comment:
do we say GA or stable in the OSS release process? I think stable but not sure sure